packages/angular_devkit/build_webpack/src/utils.ts

```diff
@@ -20,19 +21,11 @@ export interface EmittedFiles {
 export function getEmittedFiles(compilation: webpack.compilation.Compilation): EmittedFiles[] {
   const files: EmittedFiles[] = [];
 
-  // entrypoints might have multiple outputs
-  // such as runtime.js
-  for (const [name, entrypoint] of compilation.entrypoints) {
-    const entryFiles: string[] = (entrypoint && entrypoint.getFiles()) || [];
-    for (const file of entryFiles) {
-      files.push({ name, file, extension: path.extname(file), initial: true });
-    }
-  }
-
   // adds all chunks to the list of emitted files such as lazy loaded modules
-  for (const chunk of Object.values(compilation.chunks)) {
+  for (const chunk of compilation.chunks as webpack.compilation.Chunk[]) {
     for (const file of chunk.files as string[]) {
       files.push({
+        id: chunk.id.toString(),
```

Comment:
@clydin This will fail when the chunk id is `null`, since the id definition in [Chunk class](https://github.com/webpack/webpack/blob/master/lib/Chunk.js#L111-L112) tells us that it can also be `null`.

Reply:
The types are incorrect there.  It can actually be a string or a number.  At successful completion of the build an id is also required to properly construct the runtime from within Webpack.  Also of note, the name field which from the link says it is always a string, can in fact be null.